Applications/Utils/SimpleMeshCreation/generateStructuredMesh.cpp

```diff
@@ -221,6 +224,11 @@ int main (int argc, char* argv[])
     case MeshLib::MeshElemType::HEXAHEDRON:
         mesh.reset(MeshLib::MeshGenerator::generateRegularHexMesh(*vec_div[0], *vec_div[1], *vec_div[2]));
         break;
+    case MeshLib::MeshElemType::PRISM:
+        mesh.reset(MeshLib::MeshGenerator::generateRegularPrismMesh(
+            n_subdivision[0], n_subdivision[1], n_subdivision[2], vec_dx[0],
+            vec_dx[1], vec_dx[2]));
```

Comment:
Why is `vec_dx` passed here and `vec_div` in other calls?

Reply:
There doesn't exist a corresponding interface in `MeshLib/MeshGenerators/MeshGenerator.h`.

Reply:
https://github.com/ufz/ogs/blob/master/MeshLib/MeshGenerators/MeshGenerator.h#L389-L396

Reply:
Okay, you are right. There is an interface that takes the length and the number of subdivisions.

The other generator functions take `BaseLib::ISubdivision` as a parameter. An overload for the generateRegularPrismMesh that takes `BaseLib::ISubdivision` doesn't exist. So I have chosen the above function without a special reason. I'll change it to the suggested function to be a bit more consistent to the other generator functions.